src/components/IssueQueue.vue

```diff
@@ -137,18 +134,17 @@ function handleScroll() {
 
   const totalHeight = tableContainerEl.value.scrollHeight;
   const scrollPosition = tableContainerEl.value.scrollTop + tableContainerEl.value.clientHeight;
-
-  if (scrollPosition >= totalHeight) {
+  if (scrollPosition >= totalHeight - 5) {
```

Comment:
```suggestion
  if (scrollPosition >= totalHeight * 0.95) {
```

or

```suggestion
  if (scrollPosition >= totalHeight * 0.99) {
```

with a comment to explain it's to ensure load on scroll works